portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,261 @@
+  --yellow: rgb(243, 210, 80);
+  --offwhite: rgb(236, 236, 236);
+  --black: rgb(28, 33, 32);
+}
+
+@font-face {
+  font-family: Spartan;
+  src: url(./fonts/LeagueSpartan-Bold.otf);
+}
+
+@font-face {
+  font-family: HK Grotesk Light;
+  src: url(./fonts/HKGrotesk-Light.otf);
+}
+
+@font-face {
+  font-family: HK Grotesk Med;
+  src: url(./fonts/HKGrotesk-Medium.otf);
+}
+
+body {
+  background-color: white;
+}
+
+h1 {
+  color: var(--black);
+  font-family: Spartan;
+  font-size: 3rem;
+}
+
+h2 {
+  font-family: HK Grotesk Light;
+  font-size: 2rem;
+  margin-bottom: 0rem;
+}
+
+h3 {
+  font-family: HK Grotesk Med;
+  font-size: 2rem;
+  margin-bottom: 0rem;
+}
+
+p {
+  font-family: HK Grotesk Med;
+  font-size: 1rem;
+}
+
+.navbar {
+  height: 15%;
+  display: flex;
+  align-items: center;
+  padding-left: 8%;
+  padding-right: 8%;
+  box-sizing: border-box;
+  background: var(--offwhite);
+}
+
+nav {
+  flex: 1;
+  text-align: right;
+}
+
+nav ul li {
+  list-style: none;
+  display: inline-block;
+  margin-left: 60px;
+}
+
+nav ul li a {
+  text-decoration: none;
+  font-family: HK Grotesk Light;
+  font-size: 1.5rem;
+  color: var(--black);
+}
+
+nav ul li a:hover {
+  color: var(--black);
+  font-family: HK Grotesk Med;
+  text-decoration: underline;
+}
+
+button {
+  border-radius: 8px;
+  font-size: 20px;
+  font-family: HK Grotesk Light;
+  transition-duration: 0.3s;
+}
+
+button:hover {
+  background-color: var(--purple);
+  color: var(--offwhite);
+}
+
+.logo {
+  width: 100px;
+  cursor: pointer;
+}
+
+.intro {
+  padding: 8%;
+  box-sizing: border-box;
+  background-image: url(https://cdn.glitch.com/23f6eeca-cb3d-44f2-9608-0379158305f6%2Fwebsite_background.png?v=1614489189113);
+  background-position: right top;
+  background-size: cover;
+  text-align: left;
+}
+
+.intro-column {
+  float: right;
+  margin-right: 7%;
+}
+
+#about {
+  padding-left: 8%;
+  padding-right: 5%;
+  padding-top: 5%;
+  padding-bottom: 5%;
+  box-sizing: border-box;
+  background: var(--magentaPastel);
+  text-align: left;
+}
+
+#about:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.column {
+  float: left;
+  width: 50%;
+}
+
+.about-list {
+  float: left;
+}
+
+.about-list ul li {
+  list-style: none;
+  font-family: HK Grotesk Med;
+  font-size: 1rem;
+}
+
+.about-list ul li span {
+  padding-right: 5px;
+}
+
+.headshot {
+  display: block;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 70%;
+  border-radius: 8px;
+}
+
+.resume-button {
+  padding-top: 1rem;
+  text-align: center;
+}
+
+#experience {
+  padding: 8%;
+  box-sizing: border-box;
+  background: var(--purplePastel);
+  text-align: left;
+}
+
+#projects {
+  padding-left: 8%;
+  padding-right: 8%;
+  padding-bottom: 8%;
+  padding-top: 3%;
+  box-sizing: border-box;
+  background: var(--pinkPastel);
+  text-align: left;
+  height: 80rem;
+}
+
+.column h2 {
+  margin-top: 0px;
+}
+
+.project-img {
+  display: block;
+  margin-left: 10%;
```

Comment:
You might consider using the shorthand to set all of the margins at once, I think it would be this:

`margin: 10% 10% 10% auto;`

(Similar thing for padding above)

Reply:
I completely forgot that we could do that, I'll definitely change it. Thank you!